test/test_evaluation/test_train_evaluator.py

```diff
@@ -1176,6 +1221,50 @@ def test_get_splitter_cv_object(self, te_mock):
         with self.assertRaises(ValueError):
             evaluator.get_splitter(D)
 
+        # GroupKFold, regression with args
+        D.data['Y_train'] = np.array([0.0, 0.1, 0.2, 0.3, 0.4, 0.5])
+        D.info['task'] = REGRESSION
+        evaluator = TrainEvaluator()
+        evaluator.resampling_strategy = GroupKFold
+        evaluator.resampling_strategy_args = {'folds': 2, 'groups': np.array([1, 1, 2, 1, 2, 2])}
+        cv = evaluator.get_splitter(D)
+        self.assertIsInstance(cv, GroupKFold)
+        self.assertEqual(cv.get_n_splits(groups=evaluator.resampling_strategy_args['groups']), 2)
+        next(cv.split(D.data['Y_train'], D.data['Y_train'],
+                      groups=evaluator.resampling_strategy_args['groups']))
+
+        # GroupKFold, regression no args
+        D.data['Y_train'] = np.array([0.0, 0.1, 0.2, 0.3, 0.4, 0.5])
+        D.info['task'] = REGRESSION
+        evaluator = TrainEvaluator()
+        evaluator.resampling_strategy = GroupKFold
+        evaluator.resampling_strategy_args = None
+        with self.assertRaises(ValueError):
```

Comment:
Could you please use `self.assertRaisesRegex` so that the exact error is also clear and easy to understand?